src/ILCompiler.Compiler/src/Compiler/Compilation.cs

```diff
@@ -21,11 +21,23 @@ namespace ILCompiler
 {
     public class CompilationOptions
     {
-        public IReadOnlyDictionary<string, string> InputFilePaths;
-        public IReadOnlyDictionary<string, string> ReferenceFilePaths;
+        /// <summary>
+        /// Associate for each simple name, the list of assembly full path names matching the simple name.
+        /// Useful when including multiple assemblies with same name from different locations.
```

Comment:
We have essentially deprecated strong names with .NET Core. The .NET Core tooling and runtime assume that the simple name is sufficient assembly identity. We make no effort to handle same simple name / different strong name key gracefully. I do not think we should worry about it here either - it would be unnecessary complexity.

(The .NET Core tooling still checks the strong name match here and there, but just as a sanity check, e.g. to make sure that the portable assemblies still work on desktop.)